src/bundle/check.ts

```diff
@@ -0,0 +1,28 @@
+import fs from "fs";
+import path from "path";
+
+const searchInFile = (filePath: string, searchString: string) => {
+  const content = fs.readFileSync(filePath, "utf8");
+  return content.includes(searchString);
```

Comment:
This will also find the searchString if it is in a comment (and therefore never executed). I believe a more sophisticated approach is needed here like a regex. Keep in mind that it could be a line comment (//), a multiline comment (/*) or even a jsdoc comment (/**).

Reply:
Comments are stripped out in the build process, the upload command uploads your dist folder not source

Reply:
Not everyone uses a build process that removes all comments. And especially in debug builds comments have a valid use case.